src/components/players/playerDetails/playerDetails.js

```diff
@@ -32,6 +34,28 @@ const PlayerDetails = ({ player }) => {
                     <div className={styles.position}>{playerPosition}</div>
                     <div className={styles.physicalDetails}>{playerHeightFeet}-{playerHeightInches}, {playerWeight} lbs</div> 
                 </div>
+                <div className={styles.mainStatsContainer}>
```

Comment:
There is unnecessary padding, entire section not lining up flush with stats red title.

<img width="440" alt="Screen Shot 2020-03-18 at 3 16 16 AM" src="https://user-images.githubusercontent.com/623755/76935076-d257bc80-68c6-11ea-9879-9d218d355c16.png">